docs/sources/shared/datasources/tempo-traces-to-profiles.md

```diff
@@ -28,11 +28,15 @@ There are two ways to configure the trace to profiles feature:
 
 To use trace to profiles, navigate to **Explore** and query a trace. Each span now links to your queries. Clicking a link runs the query in a split panel. If tags are configured, Grafana dynamically inserts the span attribute values into the query. The query runs over the time range of the (span start time - 60) to (span end time + 60 seconds).
 
-![Selecting a link in the span queries the profile data source](/static/img/docs/tempo/profiles/tempo-profiles-Span-link-profile-data-source.png)
+{{< figure src="/static/img/docs/tempo/profiles/tempo-trace-to-profile.png" max-width="900px" class="docs-image--no-shadow" alt="Selecting a link in the span queries the profile data source" >}}
 
 To use trace to profiles, you must have a configured Grafana Pyroscope data source. For more information, refer to the [Grafana Pyroscope data source documentation](/docs/grafana/latest/datasources/grafana-pyroscope/).
 
-## Use a simple configuration
+**Embedded flame graphs** are also inserted into each span details section that has a linked profile (once you have configured a Grafana Pyroscope data source). This allows you to see resource consumption in a flame graph visualisation for each span without having to navigate away from the current view. Hover over a particular block in the flame graph to see more details about the resources being consumed.
+
+In order to determine that there is a profile for a given span and render the **Profiles for this span** button or an embedded flame graph in the span details, the ‘pyroscope.profile.id’ key-value pair must exist in your span tags.
```

Comment:
How does someone tell if the `pyroscope.profile.id` key-value pair exists in their span tags? Is this a query they can run in the Tempo data source query editor or some other easy way? 

I'm stumbling over the first section of the sentence. Is this correct? 
* If the pyorscope.profile.id key value pair is not present, then no profile data is present for a given span. 
* In that case, the Profiles for spans button is unavailable(?) and the flame graph doesn't appear. 

Is that correct? If so, would something like this be helpful:  

> If you have configured a Pyroscope data source and no profile data is available  or the Profiles for this span button is greyed out, verify that the `pyroscope.profile.id` key-value pair exists in your span tags.

Reply:
That is correct, will add the changes. Thanks, Kim :)